src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripItemCollectionTests.cs

```diff
@@ -125,6 +125,26 @@ public void ToolStripItemCollection_AddRange_ToolStripItemCollection_Success()
         Assert.Equal("c", contextMenuStrip.Items[2].Text);
     }
 
+    [WinFormsFact]
+    public void ToolStripItemCollection_AddRange_ToolStripItemCollection_WithItems_Success()
```

Comment:
```suggestion
    public void ToolStripItemCollection_AddRange_ToolStripItemCollection_SameOwner_Success()
```
I think the important difference from the previous test case is that collection owner is the same and count does not decrement